package.json

```diff
@@ -40,12 +40,14 @@
     "uri-template-route": "^1.0.0"
   },
   "devDependencies": {
+    "@rdfjs/fetch": "^2.1.0",
     "@zazuko/rdf-vocabularies": "^2019.10.22",
     "alcaeus": "^1.0.0-beta.4",
     "codecov": "^3.6.5",
+    "express-as-promise": "0.0.2",
```

Comment:
Why bring this in and not use supertest?

Reply:
I saw it and initially, I tried to use it, but it gets complicated once you want to parse the triples of the response. It's doable, but the code get's more complex. Also, using the standard fetch interfaces is maybe not a bad idea as I guess the knowledge around fetch became now more common than for supertest.

Reply:
I started using supertest because you used it first 😅. Here or in another project?

I mention this because it may be a little weird to have two way for writing similar tests.

Reply:
> I agree that we should clean up and use only one approach, but I would do that in a separate PR.

Are you saying that you would remove supertest?

Reply:
From this package: yes